python/data/ArtusConfigs/Run2LegacyAnalysis/2016/et.py

```diff
@@ -283,6 +283,7 @@ def build_config(nickname, **kwargs):
   if not isData and not isEmbedded:                 config["Processors"].append( "producer:RooWorkspaceWeightProducer")
   if isEmbedded:                 config["Processors"].append( "producer:EmbeddedWeightProducer")
   if isEmbedded:                 config["Processors"].append( "producer:TauDecayModeWeightProducer")
+
```

Comment:
I suppose we are adding the TauTriggerEfficiencyProducer here once the fits are ready for 2016, right?

Reply:
Yes